### PR TITLE
feature-create-a-DB-collection(table)-for-employer-names-for-volunteer-form[QA]

### PR DESCRIPTION
**Pull request by khmdagal**
This is a:

- [x] ✨ **New feature** - new behaviour has been implemented
- [ ] 🐛 **Bug fix** - existing behaviour has been made to behave
- [ ] ♻️ **Refactor** - the behaviour has not changed, just the implementation
- [ ] ✅ **Test backfill** - tests for existing behaviour were added but the behaviour itself hasn't changed
- [ ] ⚙️ **Chore** - maintenance task, behaviour and implementation haven't changed

- **Purpose** - 
- The purpose of this PR is to get the data of the exiting employer names from the database instead of hardcoded.
- A new volunteer can add their employer's name if it is not on the list.

- **How to check** - 

1. First visit this PR and follow the instructions.
2. Run or visit this branch: **feature-create-a-DB-of-employer-names-for-volunteer-form** in **cyf-form** repo.
3. On the 'Where you heard about CodeYourFuture' option, select employer. 
4. A new option will appear underneath. 
5. You should see the list of existing employer names and should be able to add a new employer name.

### Links

issue: https://github.com/CodeYourFuture/tech-team/issues/697
cyf-api: https://github.com/CodeYourFuture/cyf-api/pull/620

### Author checklist

- [x] I have written a title that reflects the relevant ticket
- [x] I have written a description that says what the PR does and how to validate it
- [x] I have linked to the project board ticket (and any related PRs/issues) in the Links section
- [x] I have added a link to this PR to the ticket
- [x] I have made the PR to `**QA**` from a branch named `**feature-create-a-DB-collection(table)-for-employer-names-for-volunteer-form**`
- [ ] I have completed the manual tests [described here](https://github.com/CodeYourFuture/tech-team/wiki/3.-How-we-work#manual-test-procedures)
- [x] I have requested reviewers here and in my team chat channel

- [ ] I have spoken with my PM or TL about any parts of this task that may have become out-of-scope, or any additional improvements that I now realise may benefit my project
- [ ] I have added tests, or new tests were not required
- [ ] I have updated any documentation (e.g. diagrams, schemas), or documentation updates were not required

### CODE REVIEW

**Review by AL-ASHMALI — APPROVED** [review]
Data is no longer hardcoded, instead it is coming from the employers collection in the database. The list of employers is rendered correctly on the CYF-forms

File: e2e/integration/journey.test.js
Comment (enderta) [inline e2e/integration/journey.test.js:1]: Rabie, are these cypress tests working? did you get any version issues?

**Review by enderta — APPROVED** [review]
I wonder why did you use Cypress

File: e2e/integration/journey.test.js
Comment (AL-ASHMALI) [reply e2e/integration/journey.test.js:1]: Hi Ender, it is @khmdagal who worked on the ticket, so I don't know why he used Cypress. (by the looks of it I think it was there already and @khmdagal only changed it)

Comment (enderta) [reply e2e/integration/journey.test.js:1]: did you run these tests

Comment (AL-ASHMALI) [reply e2e/integration/journey.test.js:1]: yes and it works

**Comment by esma-g** [comment]
Ready for @CanaryWharf

File: e2e/integration/journey.test.js
Comment (khmdagal) [reply e2e/integration/journey.test.js:1]: Hi @enderta the cypress testing was there and the team has been using it as long as I know, what I did in this file was just view type error corrections as @AL-ASHMALI said. Thanks, Rabie and Enderta.

**Pull request merged**